lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -242,6 +242,17 @@ def test_scatter3d_color():
                color='b', marker='s')
 
 
+@mpl3d_image_comparison(['scatter3d_depthshade_false.png'])
+def test_scatter3d_depthshade_false():
+    """
+    Test that 3d scatter plot works with depthshade=False (issue #18037)
```

Comment:
We try to be stingy with our image tests.  If this is just a test to see if depthshade=False works then maybe if doesn't need the image comparison (faster, takes less disk space etc)?

Reply:
I'll work on changing this. Thanks for the feedback!

Reply:
... you are allowed to claim it does need an image test, but no need to include one if its not necessary...

Reply:
I think this might need an image test. I can't think of another way to generate a baseline figure without using the method being tested

Reply:
You can always generate a figure, but not check the image output (just don't add the `mpl3d_image_comparison` decorator). Since the original bug resulted in a traceback, that would be enough to test that the bug is fixed.

Reply:
@neilZon You don't need to compare figures either. You can just do 
```python
     x = y = z = np.arange(16)
     ax_test = fig_test.add_subplot(projection='3d')
     ax_test.scatter(x, y, z, depthshade=False)
```
and the figure will get made.  If the code you are testing throws an error, the test will fail.  If not, then it will pass.  We call this a "smoketest" because it just checks that the code runs, and nothing else.  But bear in mind this test would fail on master right now, so its still a worthwhile (and quick!) test.

Reply:
Oh ok, I understand now. Thank you for the guidance!

Reply:
I can't seem to get my code to fail now with this test